### ISSUE TITLE
 Uncaught TypeError: Cannot read property 'connectionParameters' of undefined

### ISSUE THREAD

**Issue by niva-vilua** [vitaly-t/pg-promise#748]
I have this below snippet, which I tried to create. It seems like because of recent upgrade of pg npm package (8.3.2), this has been creating problem

```
async executeAny(db, query) {
                try {
                        return db.any(query)
                }
                catch (e) {
                        console.log("Could not execute ", query)
                        throw e
                }
        
}
```
### Expected behavior
Should return proper output for the query

### Actual behavior

```
Uncaught TypeError: Cannot read property 'connectionParameters' of undefined
      at /home/gitlab-runner/builds/tovhxyJG/0/muc/vdwh/npm-packages/db/node_modules/pg/lib/client.js:220:16
      at Object.onEnd (node_modules/pgpass/lib/helper.js:122:9)
      at Object.EventEmitter.emit (domain.js:486:12)
      at drain (node_modules/through/index.js:34:23)
      at Object.stream.queue.stream.push (node_modules/through/index.js:45:5)
      at Object.<anonymous> (node_modules/split/index.js:61:10)
      at _end (node_modules/through/index.js:65:9)
      at Object.stream.end (node_modules/through/index.js:74:5)
      at Object.onLine (node_modules/pgpass/lib/helper.js:116:24)
      at Object.EventEmitter.emit (domain.js:486:12)
      at drain (node_modules/through/index.js:36:16)
      at Object.stream.queue.stream.push (node_modules/through/index.js:45:5)
      at emit (node_modules/split/index.js:37:14)
      at next (node_modules/split/index.js:49:7)
      at Object.<anonymous> (node_modules/split/index.js:54:5)
      at Object.stream.write (node_modules/through/index.js:26:11)
      at ReadStream.ondata (_stream_readable.js:713:22)
      at ReadStream.EventEmitter.emit (domain.js:486:12)
      at addChunk (_stream_readable.js:303:12)
      at readableAddChunk (_stream_readable.js:279:9)
      at ReadStream.Readable.push (_stream_readable.js:218:10)
      at internal/fs/streams.js:226:14
      at FSReqCallback.wrapper [as oncomplete] (fs.js:528:5)
```

### Steps to reproduce
1. install latest package of the pg-promise.
2. use any function any or one, and it will throw above error

### Environment

* Version of pg-promise: 10.6.0
* OS type (Linux/Windows/Mac): Linux
* Version of Node.js: 14.8.0

I think, it is a fault from pg package. Downgrading the pg package to 8.2.2 resolved the problem. 

@beahl -- For your reference

**Comment by vitaly-t** [comment]
You need to provide more details, and an example that shows the issue.

The example you included so far tells me nothing, it is the same as saying that the library doesn't work at all, whereas all tests included into the library indicate on the contrary.

**Comment by vitaly-t** [comment]
You can see [here](https://travis-ci.org/github/vitaly-t/pg-promise/jobs/719846755) that the library v10.6.0 runs all tests successfully under Linux with NodeJS 14.8.0

**Comment by niva-vilua** [comment]
I saw this CI pipeline already. But currently I just have this information, where I am trying to execute the query and it throws above error which I mentioned and it resolves by downgrading the pg version.

**Comment by vitaly-t** [comment]
Well, this is not enough information to proceed. All my tests are working here.

**Comment by vitaly-t** [comment]
[This one seems to be the culprit](https://github.com/brianc/node-postgres/issues/2319).

**Comment by niva-vilua** [comment]
Exactly!

**Comment by vitaly-t** [comment]
Fixed with [10.6.1](https://github.com/vitaly-t/pg-promise/releases/tag/10.6.1) update.
